README.md

```diff
@@ -106,10 +109,15 @@ The French locale is built from `source/locale/fr` with the following command:
 ```commandline
 make -e SPHINXOPTS="-D language='fr'" BUILDDIR="build/fr" html
 ```
+```
+in windows: sphinx-build source build/fr -D language='fr' -b html
```

Comment:
Same as above: please separate description "in windows" and code `sphinx-build source build/fr -D language='fr' -b html`: Only the code should be in the code box.

Reply:
changed - now identical to the command directly above